client/my-sites/importer/importing-pane.jsx

```diff
@@ -194,9 +195,22 @@ export default React.createClass( {
 	},
 
 	render: function() {
-		const { site: { ID: siteId, name: siteName, single_user_site: hasSingleAuthor } } = this.props;
-		const { importerId, errorData = {}, customData } = this.props.importerStatus;
+		const {
+			importerStatus: {
+				importerId,
+				errorData = {},
+				customData
+			},
+			mapAuthor: mapAuthorFor,
```

Comment:
My gut says it's more confusing to rename. That said, I'm not familiar with the domain and may be overlooking some aspects. I say either use `mapAuthor` here, or rename the member in `mapDispatchToProps` to `mapAuthorFor`, though the former _seems_ better.